constants.js

```diff
@@ -54,7 +55,7 @@ exports.helpMessage = 'Hi! I am @' + config.get('botName')
 + " and then in the NIT KKR alumni group ask me to introduce you."
 + "\n\nBot Development credits: Saumye Malhotra (@algoro) and Devender Yadav (@itsmedev)"
 + "\n\nList of commands:"
-+ "\n/mybatchmates - gives a list of your batchmates - Developer credits: Narendra Kumawat"
++ "\n/mybatchmates - gives a list of your batchmates - Developer credits: Narendra Kumawat (@nk_kumawat)"
```

Comment:
@nkkumawat Not sure how this change is showing in your commits, while this is the change I made.
Please don't force push, it will cheat git tracking. There are no changes in this file.

Reply:
This mistake is done when squashing commits. I will not push forcefully again.

Reply:
yes sir this is done by force push.